app/controllers/users_controller.rb

```diff
@@ -12,11 +12,11 @@ def create
     @user = User.new(user_params)
     if @user.save
       reset_session
-      log_in @user 
+      log_in(@user) 
       flash[:success] = 'Welcome to our restaurant!'
```

Comment:
I can't see this rendering anything when testing on heroku.🤔 Maybe you could keep it and implement the flash later but for now I don't think is necessary.